pkg/mail/server.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information
+
+package mail
+
+import (
+	"crypto/tls"
+	"net"
+	"net/smtp"
+
+	"github.com/zeebo/errs"
+)
+
+// SMTPServer is smtp server
+type SMTPServer struct {
+	Host, Port string
+	Sender     Sender
+}
+
+// Address returns server's network address
+func (server *SMTPServer) Address() string {
+	return net.JoinHostPort(server.Host, server.Port)
+}
+
+// SendEmail sends email message to the given recipient
+func (server *SMTPServer) SendEmail(rcpt string, msg []byte) error {
```

Comment:
We probably should take an `Message` type instead of `msg []byte` since things need to be formatted as RFC 822-style email, and it's probably easier to implement that with something like:

```
type Message struct {
	From    string
	To      string
	Subject string
	Id      string
	Date    time.Time

	ReceiptTo   []string
	ContentType string

	Body string
	Parts []Part
}

type Part struct {
	Type        string
	Encoding    string
	Disposition string
	Content     string
}
```